store/task.go

```diff
@@ -197,38 +197,38 @@ func (s *TaskService) createTask(ctx context.Context, tx *sql.Tx, create *api.Ta
 	defer row.Close()
 
 	row.Next()
-	var task api.Task
+	var taskRaw api.TaskRaw
 	var databaseID sql.NullInt32
-	task.TaskRunList = []*api.TaskRun{}
-	task.TaskCheckRunList = []*api.TaskCheckRun{}
+	taskRaw.TaskRunRawList = []*api.TaskRunRaw{}
```

Comment:
We don't need to set them, as they're default to "nil".

Reply:
Fixed.